### ISSUE TITLE
Redux-persist rehydrate error after clean install

### ISSUE THREAD

**Issue by wvleeuwen** [infinitered/ignite#1227]
After clean install, immediately after the LaunchScreen is visible on screen, the red error screen comes up saying: 

```
console.error: "redux-persist: rehydrate for "primary" called after timeout.", {"github":{"avatar":"https://avatars0.githubusercontent.com/u/997157?v=4","fetching":false,"error":null,"username":"GantMan"},"_persist":{"version":-1,"rehydrated":true}}, undefined
error
    YellowBox.js:71:16
_rehydrate
    persistReducer.js:74:76
<unknown>
    persistReducer.js:97:21
tryCallOne
    core.js:37:14
<unknown>
    core.js:123:25
<unknown>
    JSTimers.js:301:23
_callTimer
    JSTimers.js:154:6
_callImmediatesPass
    JSTimers.js:202:17
callImmediates
    JSTimers.js:470:11
__callImmediates
    MessageQueue.js:278:4
<unknown>
    MessageQueue.js:145:6
__guard
    MessageQueue.js:265:6
flushedQueue
    MessageQueue.js:144:17
invokeCallbackAndReturnFlushedQueue
    MessageQueue.js:140:11
```

I have tried removing the github redux, but that simply makes the JSON description in the error message say 
```
console.error: "redux-persist: rehydrate for "primary" called after timeout.", {"_persist":{"version":-1,"rehydrated":true}}, undefined
```

Note: when redux-persist is set to `active: false`, the error does not appear.

---
Steps to reproduce

- Clean install using `ignite new [projectname]` and saying yes to everything except the dev screens.
- `yarn install`
- Fix issue [1225](https://github.com/infinitered/ignite/issues/1225) by following [these directions](https://reactnavigation.org/docs/redux-integration.html).
- `react-native run-android`

---
`ignite doctor` results:

System
  platform           darwin
  arch               x64
  cpu                8 cores                                            Intel(R) Core(TM) i7-4771 CPU @ 3.50GHz
  directory          /Users/[username]/Documents/Projects/g2m/GoToMarket

JavaScript
  node               8.5.0        /Users/[username]/.nvm/versions/node/v8.5.0/bin/node
  npm                5.3.0        /Users/[username]/.nvm/versions/node/v8.5.0/bin/npm
  yarn               1.3.2        /Users/[username]/.nvm/versions/node/v8.5.0/bin/yarn

React Native
  react-native-cli   2.0.1
  app rn version     0.50.4

Ignite
  ignite             2.0.0        /Users/[username]/.nvm/versions/node/v8.5.0/bin/ignite

Android
  java               1.8.0_101    /usr/bin/java
  android home       -            undefined

iOS
  xcode              9.2

**Comment by xemasiv** [comment]
thanks @cheddarwhizzy

**Comment by wvleeuwen** [comment]
The issue seems to have been resolved in somewhere after 5.7.1.

I noticed that my redux-persist package was on 5.7.1, where 5.9.1 was the latest. After a `yarn upgrade redux-persist`, the issue did not show up anymore. (I am running react-native 0.50.4, by the way)

**Comment by skellock** [comment]
Seems like it was a version incompatibility? Thx @wvleeuwen .
